packages/terra-consumer-nav/src/components/nav-burger-button/NavBurgerButton.scss

```diff
@@ -6,6 +6,7 @@
     border: var(--terra-consumer-burger-border, 0);
     box-shadow: var(--terra-consumer-burger-box-shadow, 0);
     color: var(--terra-consumer-burger-color, #fff);
+    font-size: 20px;
```

Comment:
should this be theme-able?

Reply:
we can always add themable options later. I would rather keep them to a minimum unless we know we need it